apps/src/templates/PaneHeader.jsx

```diff
@@ -41,6 +41,15 @@ var styles = {
     marginLeft: 3,
     marginRight: 0
   },
+  headerButtonMinecraft: {
+    backgroundColor: '#606060',
+    color: '#BFBFBF',
```

Comment:
*shrug* I'd be cool with extracting them to constants within this file, if they're only reused here.